reviews/utils/review_actions.py

```diff
@@ -245,14 +245,17 @@ def is_available(self):
             return False
 
         if (
-            review.proposal.embargo == True
+            review.proposal.embargo_end_date is not None
```

Comment:
As far as I know we don't reset `embargo_end_date` when an applicant sets `embargo` back to `False`. So there are potentially proposals with no embargo but yes embargo_end_date.

Therefore you must check both here.